java/ql/test/query-tests/EmptyMethod/Test.java

```diff
@@ -0,0 +1,39 @@
+import org.aspectj.lang.annotation.Pointcut;
+
+public class Test {
+
+  public void f() {
```

Comment:
Nit: Could we keep the compliant/non-compliant comments in the tests? I find it easier to read the test cases with the comments even if they're not needed for the inline expectations test.

```suggestion
  // COMPLIANT
  public void f() {
```